echion/config.h

```diff
@@ -117,3 +121,9 @@ set_pipe_name(PyObject *Py_UNUSED(m), PyObject *args)
 
     Py_RETURN_NONE;
 }
+
+// ----------------------------------------------------------------------------
+void _set_max_frames(unsigned int new_max_frames)
```

Comment:
does this need to be exposed to python?

Reply:
Updated to have just have Python interface